### PR TITLE
Pelion-Ready branding review

### PR DESCRIPTION
**Pull request by melwee01**

### CODE REVIEW

File: docs/device-management-test.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <h2 id="device-management-test">Validation and testing</h2>
 
-Mbed Device Management provides built-in tests to help you when define your device management configuration. Before running these tests, we recommend you refer to the [testing setup](#testing-setup) section below. 
+Device Management provides built-in tests to help you when define your device management configuration. Before running these tests, we recommend you refer to the [testing setup](#testing-setup) section below.
```
Comment (melwee01) [inline docs/device-management-test.md]: Any way to rephrase this so it's not repetitive?

File: docs/device-management-test.md [hunk 2]
```diff
@@ -57,7 +57,7 @@ Mbed Device Management tests rely on the Python SDK to test the end-to-end solut
 
 ### Testing setup
 
-1. Import an example application for Pelion Device Management that contains the corresponding configuration for your target. 
+1. Import an example application for Device Management that contains the corresponding configuration for your target.
 
    Please refer to the following [application example](https://github.com/ARMmbed/pelion-ready-example). It demonstrates how to connect to the Pelion IoT Platform service, register resources and get ready to receive a firmware update.
```
Comment (melwee01) [inline docs/device-management-test.md]: Is this really meant to be a reference to the entire platform? That include services outside Device Management.

File: docs/device-management-test.md [hunk 3]
```diff
@@ -57,7 +57,7 @@ Mbed Device Management tests rely on the Python SDK to test the end-to-end solut
...
```
Comment (melwee01) [inline docs/device-management-test.md:63]: Line 64: Same as note above.

File: docs/device-management.md [hunk 4]
```diff
@@ -94,7 +94,7 @@ This creates your private and public key pair and also initialize various `.c` f
 
 To help you start quickly, please refer to the following [application example](https://github.com/ARMmbed/pelion-ready-example). It demonstrates how to connect to the Pelion IoT Platform service, register resources and get ready to receive a firmware update.
 
-Also, there are a number of board-specific applications that focus on providing more elaborate hardware features with Mbed OS and the Pelion IoT Platform. These are available in the Pelion [quick start](https://cloud.mbed.com/quick-start). Please see the reference table below, organized by vendor name, for details:
+Also, there are a number of board-specific applications that focus on providing more elaborate hardware features with Mbed OS and the Pelion IoT Platform. These are available in the Device Management [quick start](https://cloud.mbed.com/quick-start). Please see the reference table below, organized by vendor name, for details:
```
Comment (melwee01) [inline docs/device-management.md]: Again on lines 95/97: does this include all of Pelion or just Device Management as a service?

File: docs/device-management-test.md [hunk 5]
```diff
@@ -107,7 +107,7 @@ Occasionally, if the test failed during a previous attempt, the SMCC Greentea te
 
 #### Device identity is inconsistent
 
-If your device ID in Pelion Device Management is inconsistent over a device reset, it could be because it is failing to open the credentials on the storage held in the Enhanced Secure File System. Typically, this is because the device cannot access the Root of Trust stored in SOTP.
+If your device ID in Device Management is inconsistent over a device reset, it could be because it is failing to open the credentials on the storage held in the Enhanced Secure File System. Typically, this is because the device cannot access the Root of Trust stored in SOTP.
```
Comment (melwee01) [inline docs/device-management-test.md]: "Over"?

File: docs/device-management-test.md (hunk 1 above)
Comment (iriark01) [reply docs/device-management-test.md]: I think the "help you when define" is the real issue

File: docs/device-management-test.md (hunk 2 above)
Comment (AnotherButler) [reply docs/device-management-test.md]: Do you mean the term "Pelion IoT Platform service" includes services outside Device Management, or the example does? If the former, probably not. What should it be instead?

File: docs/device-management-test.md (hunk 5 above)
Comment (AnotherButler) [reply docs/device-management-test.md]: Maybe "... is inconsistent when your device resets, it ..."?

File: docs/device-management.md [hunk 6]
```diff
@@ -1,18 +1,18 @@
...
-- Provides LWM2M resources, variables that sync automatically through Pelion Device Management.
-- Helps users avoid doing blocking network operations in interrupt contexts, by automatically defering actions to a separate thread.
-- Provides end-to-end Greentea tests for Pelion Device Management.
+- Provides LWM2M resources, variables that sync automatically through Device Management.
+- Helps users avoid doing blocking network operations in interrupt contexts, by automatically deferring actions to a separate thread.
+- Provides end-to-end Greentea tests for Device Management.
 
-This library makes it trivial to expose sensors, actuators and other variables to a cloud service. For a complete Pelion Device Management Client API, please see our [documentation](https://cloud.mbed.com/docs/current/mbed-cloud-client/index.html).
+This library makes it trivial to expose sensors, actuators and other variables to a cloud service. For a complete Device Management Client API, please see our [documentation](https://cloud.mbed.com/docs/current/mbed-cloud-client/index.html). <!---This isn't a link to API references, though.--->
```
Comment (AnotherButler) [inline docs/device-management.md]: Could you suggest a link in its place?

File: docs/device-management-test.md (hunk 2 above)
Comment (melwee01) [reply docs/device-management-test.md]: Pelion IoT Platform includes services outside of Device Management. Does the examples include/use those outside services?

Comment (melwee01) [reply docs/device-management-test.md]: If not, it should just be Device Management.

File: docs/device-management-test.md (hunk 5 above)
Comment (melwee01) [reply docs/device-management-test.md]: Sure.

File: docs/device-management.md (hunk 6 above)
Comment (melwee01) [reply docs/device-management.md]: https://cloud.mbed.com/docs/current/client-api-references/index.html

**Comment by AnotherButler** [comment]
I've made these changes manually to avoid the merge conflicts, so I'm closing this PR.

**Comment by iriark01** [comment]
Sorry, that was my fault

**Pull request closed without merging**